### DISCUSSION TITLE
Is it possible to set the interval in hours?

### DISCUSSION THREAD

**Discussion by Arm0ID** [st3v3nmw/obsidian-spaced-repetition#1181] [Q&A] [answered]
I would like to set the interval like this: 
**difficult**- 3 hours | **normal** - 1 day | **easy** - 2 days
But I read about the algorithm and didn’t understand how to do it. There are suspicions that we need to wait for an update, but maybe there is some way to rewrite a couple of lines of code?

**Comment by alejsm** [comment]
Hello, I have the same kind of problem.

I have been previously a user of Anki. In Anki there is an interval of, if I remember well, 15 minutes. So if you fail to answer the card, then the card is repeated in the same day until you mark it as learned and then it will marked for review in 1 or 2 days.

With spaced repetition it is more difficult to learn hard cards because you have to wait until the next day to review it.

**Comment by 4Source** [comment, marked as answer]
At the moment there is no way to get such short intervals. But Maybe a feature you didn't know about is the cram It is not quite the same but maybe you like it. It allows you to Repeat Flashcards in a Deck or in a Note till you mark it as easy (only wrong or easy possible), but it would not update your ease values from normal Review, means you those cards won't be marked as learned today. 

You can access it by command: 
- Spaced Repetition: Cram flashcards in this note
- Spaced Repetition: Select a deck to cram

**Reply by alejsm** [reply]
Thanks, I will try the cram feature!

**Comment by KyleKlus** [comment]
For all updates on this feature please visit this #1397 issue 😊
